style.css

```diff
@@ -0,0 +1,967 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+@import url("./style_utilities/variables.css");
+@import url("./style_utilities/utilities.css");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto";
+  text-decoration: none;
+}
+
+a {
```

Comment:
Try to add CSS to class so if anyone wants to use that HTML component in pure form then they can use

Reply:
ok will keep in mind for the future update. Thanks for suggesting